getting-started/installation-apt-ubuntu.md

```diff
@@ -20,15 +18,24 @@ If you wish to maintain your current version of PostgreSQL outside
 of `apt`, we recommend installing from source.  Otherwise, please be
 sure to remove non-`apt` installations before using this method.
 
+**If you don't already have PostgreSQL installed**, add PostgreSQL's third
+party repository to get the latest PostgreSQL packages:
+```bash
+# `lsb_release -c -s` should return the correct codename of your OS
+echo "deb http://apt.postgresql.org/pub/repos/apt/ `lsb_release -c -s`-pgdg main" >> /etc/apt/sources.list.d/pgdg.list
+wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add - \
+sudo apt-get update
+```
+
 ```bash
 # Add our PPA
 sudo add-apt-repository ppa:timescale/timescaledb-ppa
 sudo apt-get update
 
-# To install for PG 9.6.3+
-sudo apt install timescaledb-postgresql-9.6
 # To install for PG 10.2+
 sudo apt install timescaledb-postgresql-10
+# To install for PG 9.6.3+
+sudo apt install timescaledb-postgresql-9.6
```

Comment:
Got this error on Ubuntu 14.04

The following packages have unmet dependencies:
 timescaledb-postgresql-9.6 : Depends: timescaledb-loader-postgresql-9.6 (>= 0.12.1~) but it is not going to be installed

Reply:
Hmm interesting, wonder if I forgot to rebuild for 14.04. Will double check on my machine.